fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanTransaction.java

```diff
@@ -127,9 +127,15 @@ public class LoanTransaction extends AbstractAuditableWithUTCDateTimeCustom {
     @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY, mappedBy = "loanTransaction")
     private Set<LoanCollateralManagement> loanCollateralManagementSet = new HashSet<>();
 
+    @Column(name = "is_full_chargeback", nullable = false)
+    private boolean fullChargeback;
```

Comment:
@josehernandezfintecheandomx Please do not introduce this field... if you need this information we can calculate it from the transactions, but for most of the transactions this field is irrelevant..

Reply:
Done, removed